### PR TITLE
fixes typo in readme

### PR DESCRIPTION
**Pull request by manumaticx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ var os = require('os');
 var path = require('path');
 var port = 8888;
 
-var av = require('./');
+var av = require("tessel-av");
```
Comment (LinusU) [inline README.md]: Can we keep the single quotes?

Comment (manumaticx) [inline README.md]: Sure. Will update the above example too (copied from there)

**Comment by HipsterBrown** [comment]
LGTM. Thanks for catching that @manumaticx

**Review by LinusU — APPROVED** [review]
Stellar ✨

**Pull request merged**